python/cuml/benchmark/runners.py

```diff
@@ -291,8 +299,6 @@ def _run_one_size(
             for rep in cpu_timer.benchmark_runs():
                 cpu_model = algo_pair.run_cpu(
                     data,
-                    **param_overrides,
-                    **cpu_param_overrides,
```

Comment:
Passing `param_overrides` and `cpu_param_overrides` causes and error if those parameters are actually present. This is because they already copied to the `setup_override`.